code/modules/projectiles/guns/projectile/contender.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/weapon/gun/projectile/contender
+	name = "Thompson Contender"
+	desc = "A perfect, pristine replica of an ancient one-shot hand-cannon. For when you really want to make a hole. This one has been modified to work almost like a bolt-action."
+	icon_state = "pockrifle"
+	var/icon_retracted = "pockrifle-empty"
+	item_state = "revolver"
+	caliber = "357"
```

Comment:
This is supposed to be ".357" but I forgot to fix that in #3475 so remind me to do that after this gets merged.